portfolio/src/main/webapp/index.html

```diff
@@ -6,11 +6,11 @@
     <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="font-awesome-styles.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <!-- Portafolio Container -->
+    <div id="page_container">
```

Comment:
Style guide: Separate words in ID and class names by a hyphen.
I think you can keep like this, but try to be consistent, I've seen other IDs with hyphens instead of underscores. :)

Reply:
Thanks for pointing it out, I haven' saw that!